charts/dev/livebook/templates/_secrets.tpl

```diff
@@ -10,6 +10,8 @@
 {{- end }}
 enabled: true
 data:
-  LIVEBOOK_SECRET_KEY_BASE: {{ $secretKeyBase }}
-  LIVEBOOK_COOKIE: {{ $cookie }}
+  LIVEBOOK_SECRET_KEY_BASE: {{ $secretKeyBase | quote }}
+  LIVEBOOK_COOKIE: {{ $cookie | quote }}
+  LIVEBOOK_PASSWORD: {{ .Values.livebookPassword | quote }}
+  AWS_SECRET_ACCESS_KEY: {{ .Values.awsSecretAccessKey | default "false" | quote }}
```

Comment:
```suggestion
  AWS_SECRET_ACCESS_KEY: {{ .Values.awsSecretAccessKey | default false | quote }}
```
quote should handle that already
but are we sure we want to put `"false"` when there is not key given? 
Maybe empty is more apropriate?

Reply:
You are right. Empty makes more sense...